Servers/utils/project.utils.ts

```
@@ -481,16 +489,18 @@ export const deleteHelper = async (
     childObject[childTableName].foreignKey,
     parent_id,
+    tenant,
     transaction
   );
 };
 
 export const deleteProjectByIdQuery = async (
   id: number,
+  tenant: string,
   transaction: Transaction
 ): Promise<Boolean> => {
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)